docs/src/pages/index.tsx

```diff
@@ -6,41 +6,274 @@ import clsx from "clsx";
 // There should be a line here that says
 // import React from "react";
 // VSCode might try to delete it, but that will break the site.
-import React from "react";
+import React, { useState } from "react";
 import styles from "./index.module.css";
+import '../css/style.css'
 
 function HomepageHeader() {
   const { siteConfig } = useDocusaurusContext();
+  const [isActive, setIsActive] = useState(false);
+  const toggleMenu = () => {
+    setIsActive((prevState) => !prevState);
+  };
+
+
   return (
-    <header className={clsx("hero hero--primary", styles.heroBanner)}>
-      <div className="container">
-        <h1 className="hero__title">{siteConfig.title}</h1>
-        <p className="hero__subtitle">{siteConfig.tagline}</p>
-        <div className={styles.buttons}>
-          <Link
-            className="button button--secondary button--lg"
-            to="/docs/getting-started"
-          >
-            Tutorial - ⏱️ 5 min
-          </Link>
+      <header className="header">
+        <div className="container">
+          <div className="header-wrapper">
+            <a href="" className="logo"><img src="img/dev_logo_dark.svg" alt="logo"/></a>
+            <ul className={`menu ${isActive ? "active" : ""}`}>
+              <li className="contain-logo"><a href="" className="logo"><img src="img/logo.png" alt=""/></a></li>
+              <li className="menu-item"><a className="menu-link" href="https://code.groundlight.ai/python-sdk/docs/getting-started">Docs</a></li>
```

Comment:
is it possible to just use the relative paths here and everywhere else for https://code.groundlight.ai links?
e.g. "python-sdk/docs/getting-started" instead of "https://code.groundlight.ai/python-sdk/docs/getting-started".

This will make the page work even on a local development server.